src/common/offline_transformations/CMakeLists.txt

```diff
@@ -20,14 +20,18 @@ source_group("include" FILES ${PUBLIC_HEADERS})
 add_library(${TARGET_NAME} STATIC EXCLUDE_FROM_ALL ${LIBRARY_SRC} ${PUBLIC_HEADERS})
 
 target_link_libraries(${TARGET_NAME} PUBLIC ngraph inference_engine_transformations ngraph::reference
-                                            nlohmann_json_schema_validator
-                                     PRIVATE openvino::itt pugixml::static)
+                                            nlohmann_json_schema_validator nlohmann_json
```

Comment:
looks like `nlohmann_json_schema_validator nlohmann_json` are not `PUBLIC` dependencies anymore (not used in public header files)

Reply:
yes, you are right 
fixed